### PR TITLE
Task 1: Static view of index page

### PR DESCRIPTION
**Pull request by MateuszKubuszok**
screenshot from 2016-08-30 11 17 12

### CODE REVIEW

File: src/components/Main.js [hunk 1]
```diff
@@ -2,15 +2,24 @@ require('normalize.css/normalize.css');
 require('styles/App.css');
```
Comment (burczu) [inline src/components/Main.js]: should be changed to `import 'styles/App.css'`

File: src/components/Main.js [hunk 2]
```diff
@@ -2,15 +2,24 @@ require('normalize.css/normalize.css');
...
 
 import React from 'react';
-
-let yeomanImage = require('../images/yeoman.png');
+import Recipes from '../components/Recipes';
 
 class AppComponent extends React.Component {
   render() {
+    let recipesRepo = new Recipes
```
Comment (burczu) [inline src/components/Main.js]: use `const` if using `let` is not necessary

File: src/components/Main.js [hunk 3]
```diff
@@ -2,15 +2,24 @@ require('normalize.css/normalize.css');
...
+    let recipes = recipesRepo.search()
```
Comment (burczu) [inline src/components/Main.js]: lack of semicolons!!

File: src/components/Recipes.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+class Recipes {
```
Comment (burczu) [inline src/components/Recipes.js]: this class should be rather in the `src/data` folder or something like that

File: src/components/Main.js [hunk 5]
```diff
@@ -2,15 +2,24 @@ require('normalize.css/normalize.css');
...
     return (
       <div className="index">
-        <img src={yeomanImage} alt="Yeoman Generator" />
-        <div className="notice">Please edit <code>src/components/Main.js</code> to get started!</div>
+        <h1 className="title">Recipies</h1>
+        <div className="search-box"><input type="text" value="Search here..." /></div>
+        <div className="recipes-list">
+        {recipes.map((recipe) => {
+        	return <div className="recipe-wrap">
```
Comment (burczu) [inline src/components/Main.js]: put `div` into `()` and add semicolon at the end

**Comment by pjazdzewski1990** [comment]
Closing

**Pull request closed without merging**